gnomad/resources/grch38/reference_data.py

```diff
@@ -154,8 +154,22 @@ def _import_dbsnp(**kwargs) -> hl.Table:
 )
 
 dbsnp = VersionedTableResource(
-    default_version="b154",
+    default_version="b156",
     versions={
+        "b156": GnomadPublicTableResource(
+            # can't find this path anymore, is it temporary for intermediate files?
+            path="gs://gnomad-public-requester-pays/resources/grch38/dbsnp/dbsnp_b156_grch38_all_20221116.ht",
+            import_func=_import_dbsnp,
+            import_args={
+                "path": "gs://gcp-public-data--gnomad/resources/grch38/dbsnp/dbsnp_b156_grch38_all_GCF_000001405.40_20221116.vcf.bgz",
+                "header_file": "gs://gcp-public-data--gnomad/resources/grch38/dbsnp/dbsnp_b156_grch38_all_GCF_000001405.40_20221116.vcf.header",
+                "force_bgz": True,
+                "contig_recoding": DBSNP_B154_CHR_CONTIG_RECODING,
```

Comment:
Just confirming that B156 uses the same recoding as B154?

Reply:
Yes, I checked for chr1, the file is too big, that I didn't get to all the lines.